From e6dcb14dbbd3e9e40a1f193a7bf6657e82b88cb9 Mon Sep 17 00:00:00 2001
From: Nikos Mavrogiannopoulos <nmav@redhat.com>
Date: Mon, 15 Feb 2016 09:52:10 +0100
Subject: [PATCH] tests: mini-loss-time: ensure client timeouts after the
 server is

This addresses issue with the server detecting the client disconnection
prior to its timeout. Reported by Steven Chamberlain, Andreas Metzler.
---
 tests/mini-loss-time.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/mini-loss-time.c b/tests/mini-loss-time.c
index 13de21e..b95f631 100644
--- a/tests/mini-loss-time.c
+++ b/tests/mini-loss-time.c
@@ -50,7 +50,7 @@ int main()
 #include "utils.h"
 
 /* This program tests whether a DTLS handshake would timeout
- * in a minute.
+ * in the expected time.
  */
 
 static void print_type(const unsigned char *buf, int size)
@@ -136,7 +136,7 @@ static void client(int fd)
 	 */
 	gnutls_init(&session, GNUTLS_CLIENT | GNUTLS_DATAGRAM);
 	gnutls_dtls_set_mtu(session, 1500);
-	gnutls_dtls_set_timeouts(session, 1 * 1000, 30 * 1000);
+	gnutls_dtls_set_timeouts(session, 1 * 1000, 31 * 1000);
 
 	/* Use default priorities */
 	gnutls_priority_set_direct(session,
-- 
2.7.0

